manim/utils/debug.py

```diff
@@ -16,12 +20,62 @@ def print_family(mobject, n_tabs=0):
         print_family(submob, n_tabs + 1)
 
 
-def index_labels(mobject, label_height=0.15):
+def index_labels(
+    mobject: "Mobject",
+    label_height: float = 0.15,
+    background_stroke_width=5,
+    background_stroke_color=BLACK,
+    **kwargs
+):
+    r"""Returns a :class:`~.VGroup` of :class:`~.Integer`s
```

Comment:
I think adding an s right after a reference breaks the reference

```suggestion
    r"""Returns a :class:`~.VGroup` of :class:`~.Integer` mobjects
```